packages/client/src/recommendations/types/getProductRecommendations.types.ts

```diff
@@ -0,0 +1,50 @@
+import type { Config } from '../../types';
+
+export type GetProductRecommendations = (
+  query: GetProductRecommendationQuery,
+  config?: Config,
+) => Promise<GetProductRecommendation[]>;
+
+export type GetProductRecommendationQuery = {
+  productId: number;
+  strategyName: string;
+  howMany?: number;
+  countryCode?: string;
+  gender?: string;
+  userType?: string;
+  userIdentifier?: string;
+  categoryId?: number;
+  brandId?: string;
+  merchantId?: number;
+};
+
+export type GetProductRecommendation = {
+  id: string;
+  products: GetProductItem[];
+  recommendationTracker?: GetRecommendationTrackerItem[];
+};
+
+export enum GetRecommendationTrackerType {
+  Load,
+  View,
+  Click,
+}
+
+export enum GetProductRecommendationType {
+  Internal,
+  Sponsored,
+}
+
+export type GetRecommendationTrackerItem = {
+  type: GetRecommendationTrackerType;
+  targetUri: string;
+};
+
+export type GetProductItem = {
```

Comment:
Same as above, this should be called ProductItem and all other types that are not specific to the get request should not have the request method prefixed to the type.

Reply:
Actually never mind, I just noticed you are doing that already.